### ISSUE TITLE
Configure PHPCR ODM child restrictions

### ISSUE THREAD

**Issue by wouterj** [symfony-cmf/content-bundle#173]
We should implement https://github.com/doctrine/phpcr-odm/pull/706 in 2.0 of this bundle. The tree browser bundle can use this information to restrict children.

**Comment by wouterj** [comment]
Also skipping this one. Not sure how to do this, almost everything is allowed as child of `StaticContent`.

**Comment by dbu** [comment]
yeah, not sure if its a good idea to restrict here at all. probably should be up to the user if they need restrictions.
